package-lock.json

```diff
@@ -7739,25 +7739,25 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.1.1.tgz",
+          "resolved": false,
```

Comment:
Can we leave this `package-lock` stuff out of this commit/PR? Apparently it's a bug in NPM

Reply:
Yes, I will take it out.